e2e/tests/selectHearing_test.js

```diff
@@ -0,0 +1,36 @@
+const config = require('../config.js');
+
+Feature('Select hearing');
+
+Before((I, caseViewPage) => {
+  I.logInAndCreateCase(config.swanseaLocalAuthorityEmailUserOne, config.localAuthorityPassword);
+  caseViewPage.goToNewActions(config.applicationActions.selectHearing);
+});
+
+Scenario('completing half the fields in the Select hearing section of the c110a application', (I, caseViewPage, enterHearingNeededEventPage) => {
```

Comment:
Let's don't implement tests like that for the future :). I know Damian removed them in his PR, and I don't see them adding value, adding couple of fields extra is not adding a lot of time, so let's have just one with all the fields.

Reply:
this is not my story must have been on master, does this test suite need to be deleted? @akomisarek